env/build_jet_intel.env_DA

```diff
@@ -15,27 +15,24 @@ export CMAKE_CXX_COMPILER=mpiicpc
 export CMAKE_Fortran_COMPILER=mpiifort
 export CMAKE_Platform=jet.intel
 
-export NCEPLIBS="/lfs4/BMC/wrfruc/gge/precompiled/NCEPLIBS/b_intel18.0.5.274_impi2018.4.274/install"
-export GSILIBS="/lfs4/BMC/wrfruc/gge/precompiled/GSILIBS/b_intel18.0.5.274_impi2018.4.274"
-export BACIO_LIB4=${NCEPLIBS}/lib/libbacio_4.a
-export  BUFR_LIBd=${GSILIBS}/lib/libbufr_v.a
-export GSIWRF_LIB=${GSILIBS}/lib/libWRFLIB.a
-export   CRTM_LIB=${NCEPLIBS}/lib/libcrtm.a
-export   CRTM_INC=${NCEPLIBS}/include
-export NEMSIO_LIB=${NCEPLIBS}/lib/libnemsio.a
-export NEMSIO_INC=${NCEPLIBS}/include
-export SFCIO_LIB4=${NCEPLIBS}/lib/libsfcio_4.a
-export SFCIO_INC4=${NCEPLIBS}/include_4
-export SIGIO_LIB4=${NCEPLIBS}/lib/libsigio_4.a
-export SIGIO_INC4=${NCEPLIBS}/include_4
-export    SP_LIBd=${NCEPLIBS}/lib/libsp_d.a
-export    SP_LIB4=${NCEPLIBS}/lib/libsp_4.a
-export W3EMC_LIBd=${NCEPLIBS}/lib/libw3emc_d.a
-export W3EMC_LIB4=${NCEPLIBS}/lib/libw3emc_4.a
-export W3EMC_INCd=${NCEPLIBS}/include_d
-export W3EMC_INC4=${NCEPLIBS}/include_4
-export W3NCO_LIBd=${NCEPLIBS}/lib/libw3nco_d.a
-export W3NCO_LIB4=${NCEPLIBS}/lib/libw3nco_4.a
-export    IP_LIBd=${NCEPLIBS}/lib/libip_d.a
-export    IP_LIB4=${NCEPLIBS}/lib/libip_4.a
+module use /lfs4/HFIP/hfv3gfs/nwprod/hpc-stack/libs/modulefiles/stack
+module load hpc/1.1.0
+module load hpc-intel/18.0.5.274
+module load hpc-impi/2018.4.274
+module load bufr/11.4.0
+module load bacio/2.4.1
+module load crtm/2.3.0
+module load ip/3.3.3
+module load nemsio/2.5.2
+module load sp/2.3.3
+module load w3emc/2.7.3
+module load w3nco/2.4.1
+module load sfcio/1.4.1
+module load sigio/2.3.2
 
+module use  /mnt/lfs4/HFIP/hfv3gfs/nwprod/NCEPLIBS/modulefiles
+module load jasper/v1.900.1
+module load png/v1.2.44
+module load z/v1.2.6
+module load g2/v3.1.0
+module load g2tmpl/v1.6.0
```

Comment:
@hu5970 I'm curious if you would be able to build DA with the env file that is in the current community develop: https://github.com/ufs-community/ufs-srweather-app/blob/develop/env/build_jet_intel.env

It seems like there is a LOT of overlap there, but the develop one has mostly newer versions. Maybe backward compatibility was maintained. There may need to be some mapping between the variables available from the hpc-stack modules and the ones expected by cmake when building DA components, but we should be able to put those in the CMakeLists.txt files. If it will build with those libraries, we would greatly simplify this build process.